backend/clubs/models.py

```diff
@@ -1896,6 +1877,66 @@ def send_confirmation_email(self):
             )
 
 
+class TicketTransactionRecord(models.Model):
+    """
+    Represents an instance of a transaction record for an ticket, used for bookkeeping
+    """
+
+    ticket = models.ForeignKey(
+        Ticket, related_name="transaction_records", on_delete=models.PROTECT
+    )
+    reconciliation_id = models.CharField(max_length=100, null=True, blank=True)
+    total_amount = models.DecimalField(max_digits=5, decimal_places=2)
+    buyer_phone = PhoneNumberField(null=True, blank=True)
+    buyer_first_name = models.CharField(max_length=100)
+    buyer_last_name = models.CharField(max_length=100)
+    buyer_email = models.EmailField(blank=True, null=True)
+
+
+class TicketTransferRecord(models.Model):
+    """
+    Represents a transfer of ticket ownership, used for bookkeeping
+    """
+
+    ticket = models.ForeignKey(
+        Ticket, related_name="transfer_records", on_delete=models.PROTECT
+    )
+    sender = models.ForeignKey(
+        get_user_model(),
+        related_name="sent_transfers",
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+    )
+    receiver = models.ForeignKey(
+        get_user_model(),
+        related_name="received_transfers",
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+    )
```

Comment:
Should these be nullable?

Reply:
If you want to use `SET_NULL`...

Reply:
lmao my bad. Was hoping there was a way to disallow null references on creation while using SET_NULL, but idk if there is.